File: CS/0771_Jewels_and_Stones.cs
public class Solution {
    public int NumJewelsInStones(string jewels, string stones) {
        int result = 0;
        for(int i = 0; i < jewels.Length; i++){
            for(int j = 0; j < stones.Length; j++){
                if(jewels[i] == stones[j]){
                    result++;
                }
            }
        }
        return result;
    }
}

File: Cpp/0771_Jewels_and_Stones.cpp
class Solution {
public:
    int numJewelsInStones(string jewels, string stones) {
        int result = 0;
        for(auto c : stones){
            if(jewels.find(c) != string::npos){
                result++;
            }
        }
        return result;
    }
};
